ios/Example/Tests/DEREncodableTest.swift

```diff
@@ -124,7 +124,8 @@ final class DEREncodableTest: XCTestCase {
     func testDERUTCTime() throws {
         // "2023-02-13T20:09:30Z"
         let value = try DERUTCTime(Date(timeIntervalSince1970: TimeInterval(1676318970)))
```

Comment:
Have you tested bad input value?

Reply:
not really since the formatter will normalize the output. a new instantiation of a Date object wouldnt change anything.